src/ssl.c

```diff
@@ -15691,6 +15719,171 @@ WOLFSSL_API long wolfSSL_set_tlsext_status_ids(WOLFSSL *s, void *arg)
     return 0;
 }
 
+/*** TBD ***/
+WOLFSSL_API int SSL_SESSION_set1_id(WOLFSSL_SESSION *s, const unsigned char *sid, unsigned int sid_len)
+{
+    (void)s;
+    (void)sid;
+    (void)sid_len;
+    return 1;
```

Comment:
For the TBD's let add the stub macro for easy ID in the future.

WOLFSSL_STUB("tbd function name");

SSL_SUCCESS should be preferred over 1, but is success really what we want to return for an unimplemented stub?